domains/oppoal.json

```
@@ -0,0 +1,14 @@
+{
+    "owner": {
+        "username": "oppoa1",
+        "email": "daniel_zhong@aliyun.com"
+    },
+    "records": [
+        {
+            "NS": "clay.ns.cloudflare.com"
+        },
+        {
+            "NS": "fay.ns.cloudflare.com"
+        }
+    ]
```

please read [the docs](https://docs.is-a.dev/).
```suggestion
    "records": {
            "NS": [ "clay.ns.cloudflare.com", "fay.ns.cloudflare.com" ]
     }
```
also you should provide a reason for ns records